codebase/superdarn/src.bin/tk/tool/find_freqband.1.0/find_freqband.c

```diff
@@ -0,0 +1,244 @@
+/* find_freqband.c
+   ===============
+   Author: Angeline G. Burrell - NRL - 2021
+*/
+
+/*
+ LICENSE AND DISCLAIMER
+
+ This file is part of the Radar Software Toolkit (RST).
+
+ RST is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Lesser General Public License as published by
```

Comment:
Did you intend to use the LGPL disclaimer here rather than GPL? See also `channel.c`

Reply:
Ok, I found an example and propagated it.  If it's wrong I'll have another go.